src/pip/_internal/utils/markers.py

```diff
@@ -0,0 +1,20 @@
+import os.path
+
```

Comment:
nitpick: two empty lines after imports section of module

Reply:
That's what I did ^^ But isort wasn't happy :man_shrugging: